### ISSUE TITLE
Cannot install libjpeg62-turbo-dev

### ISSUE THREAD

**Issue by willin** [Automattic/node-canvas#747]
apt-get install libjpeg62-turbo-dev

```
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package libjpeg62-turbo-dev
```

source: digitalocean

**Comment by LinusU** [comment]
Which version of Ubuntu/Debian?

**Comment by willin** [comment]
Linux 3.13.0-79-generic #123-Ubuntu SMP Fri Feb 19 14:27:58 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

Ubuntu 14.04.4 LTS (GNU/Linux 3.13.0-79-generic x86_64)

[digitalocean](https://m.do.co/c/bf3884bf4471)

```
Get:1 http://security.ubuntu.com trusty-security InRelease [65.9 kB]
Get:2 http://security.ubuntu.com trusty-security/main Sources [110 kB]
Get:3 http://security.ubuntu.com trusty-security/universe Sources [35.2 kB]
Get:4 http://ag-projects.com precise InRelease [6,468 B]
Get:5 http://security.ubuntu.com trusty-security/main amd64 Packages [455 kB]
Ign http://ag-projects.com precise InRelease
Ign http://ag-projects.com precise/main Sources/DiffIndex
Ign http://ag-projects.com precise/main amd64 Packages/DiffIndex
Ign http://ag-projects.com precise/main i386 Packages/DiffIndex
Ign http://mirrors.digitalocean.com trusty InRelease
Get:6 http://security.ubuntu.com trusty-security/universe amd64 Packages [126 kB]
Get:7 http://security.ubuntu.com trusty-security/main i386 Packages [429 kB]
Get:8 http://security.ubuntu.com trusty-security/universe i386 Packages [126 kB]
Get:9 http://mirrors.digitalocean.com trusty-updates InRelease [65.9 kB]
Hit http://security.ubuntu.com trusty-security/main Translation-en
Hit http://security.ubuntu.com trusty-security/universe Translation-en
Hit http://ag-projects.com precise/main Sources
Hit http://ag-projects.com precise/main amd64 Packages
Hit http://ag-projects.com precise/main i386 Packages
Ign http://ag-projects.com precise/main Translation-en_US
Ign http://ag-projects.com precise/main Translation-en
Hit http://mirrors.digitalocean.com trusty-backports InRelease
100% [7 Packages bzip2 0 B] [Waiting for headers]^Cwulian@fr:~$ sudo apt-get update
Ign http://mirrors.digitalocean.com trusty InRelease
Get:1 http://mirrors.digitalocean.com trusty-updates InRelease [65.9 kB]
Hit http://security.ubuntu.com trusty-security InRelease
Hit http://mirrors.digitalocean.com trusty-backports InRelease
Get:2 http://mirrors.digitalocean.com trusty Release.gpg [933 B]
Hit http://mirrors.digitalocean.com trusty Release
Hit http://security.ubuntu.com trusty-security/main Sources
Get:3 http://ag-projects.com precise InRelease [6,468 B]
Get:4 http://mirrors.digitalocean.com trusty-updates/main Sources [271 kB]
Hit http://security.ubuntu.com trusty-security/universe Sources
Hit http://security.ubuntu.com trusty-security/main amd64 Packages
Hit http://security.ubuntu.com trusty-security/universe amd64 Packages
Get:5 http://mirrors.digitalocean.com trusty-updates/restricted Sources [5,352 B]
Hit http://security.ubuntu.com trusty-security/main Translation-en
Get:6 http://mirrors.digitalocean.com trusty-updates/universe Sources [152 kB]
Hit http://security.ubuntu.com trusty-security/universe Translation-en
Get:7 http://mirrors.digitalocean.com trusty-updates/multiverse Sources [5,946 B]
Ign http://ag-projects.com precise InRelease
Get:8 http://security.ubuntu.com trusty-security/main i386 Packages [429 kB]
Get:9 http://mirrors.digitalocean.com trusty-updates/main amd64 Packages [754 kB]
Ign http://ag-projects.com precise/main Sources/DiffIndex
Ign http://ag-projects.com precise/main amd64 Packages/DiffIndex
Ign http://ag-projects.com precise/main i386 Packages/DiffIndex
Get:10 http://mirrors.digitalocean.com trusty-updates/restricted amd64 Packages [15.9 kB]
Get:11 http://mirrors.digitalocean.com trusty-updates/universe amd64 Packages [357 kB]
Get:12 http://security.ubuntu.com trusty-security/universe i386 Packages [126 kB]
Get:13 http://mirrors.digitalocean.com trusty-updates/multiverse amd64 Packages [13.2 kB]
Get:14 http://mirrors.digitalocean.com trusty-updates/main i386 Packages [720 kB]
Hit http://ag-projects.com precise/main Sources
Get:15 http://mirrors.digitalocean.com trusty-updates/restricted i386 Packages [15.6 kB]
Hit http://ag-projects.com precise/main amd64 Packages
Get:16 http://mirrors.digitalocean.com trusty-updates/universe i386 Packages [359 kB]
Hit http://ag-projects.com precise/main i386 Packages
Ign http://ag-projects.com precise/main Translation-en_US
Ign http://ag-projects.com precise/main Translation-en
Get:17 http://mirrors.digitalocean.com trusty-updates/multiverse i386 Packages [13.6 kB]
Hit http://mirrors.digitalocean.com trusty-updates/main Translation-en
Hit http://mirrors.digitalocean.com trusty-updates/multiverse Translation-en
Hit http://mirrors.digitalocean.com trusty-updates/restricted Translation-en
Hit http://mirrors.digitalocean.com trusty-updates/universe Translation-en
Hit http://mirrors.digitalocean.com trusty-backports/main Sources
Hit http://mirrors.digitalocean.com trusty-backports/restricted Sources
Hit http://mirrors.digitalocean.com trusty-backports/universe Sources
Hit http://mirrors.digitalocean.com trusty-backports/multiverse Sources
Hit http://mirrors.digitalocean.com trusty-backports/main amd64 Packages
Hit http://mirrors.digitalocean.com trusty-backports/restricted amd64 Packages
Hit http://mirrors.digitalocean.com trusty-backports/universe amd64 Packages
Hit http://mirrors.digitalocean.com trusty-backports/multiverse amd64 Packages
Hit http://mirrors.digitalocean.com trusty-backports/main i386 Packages
Hit http://mirrors.digitalocean.com trusty-backports/restricted i386 Packages
Hit http://mirrors.digitalocean.com trusty-backports/universe i386 Packages
Hit http://mirrors.digitalocean.com trusty-backports/multiverse i386 Packages
Hit http://mirrors.digitalocean.com trusty-backports/main Translation-en
Hit http://mirrors.digitalocean.com trusty-backports/multiverse Translation-en
Hit http://mirrors.digitalocean.com trusty-backports/restricted Translation-en
Hit http://mirrors.digitalocean.com trusty-backports/universe Translation-en
Hit http://mirrors.digitalocean.com trusty/main Sources
Hit http://mirrors.digitalocean.com trusty/restricted Sources
Hit http://mirrors.digitalocean.com trusty/universe Sources
Hit http://mirrors.digitalocean.com trusty/multiverse Sources
Hit http://mirrors.digitalocean.com trusty/main amd64 Packages
Hit http://mirrors.digitalocean.com trusty/restricted amd64 Packages
Hit http://mirrors.digitalocean.com trusty/universe amd64 Packages
Hit http://mirrors.digitalocean.com trusty/multiverse amd64 Packages
Hit http://mirrors.digitalocean.com trusty/main i386 Packages
Hit http://mirrors.digitalocean.com trusty/restricted i386 Packages
Hit http://mirrors.digitalocean.com trusty/universe i386 Packages
Hit http://mirrors.digitalocean.com trusty/multiverse i386 Packages
Hit http://mirrors.digitalocean.com trusty/main Translation-en
Hit http://mirrors.digitalocean.com trusty/multiverse Translation-en
Hit http://mirrors.digitalocean.com trusty/restricted Translation-en
Hit http://mirrors.digitalocean.com trusty/universe Translation-en
Ign http://mirrors.digitalocean.com trusty/main Translation-en_US
Ign http://mirrors.digitalocean.com trusty/multiverse Translation-en_US
Ign http://mirrors.digitalocean.com trusty/restricted Translation-en_US
Ign http://mirrors.digitalocean.com trusty/universe Translation-en_US
Fetched 3,312 kB in 8s (369 kB/s)
Reading package lists... Done
W: GPG error: http://ag-projects.com precise InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 07C0EF03163A0DF5
```

**Comment by willin** [comment]
ubuntu@cam-api-1:~/dep/node-v6.1.0$ sudo apt-get install libcairo2-dev libjpeg62-turbo-dev libpango1.0-dev libgif-dev build-essential g++
sudo: unable to resolve host cam-api-1
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package libjpeg62-turbo-dev

**Comment by willin** [comment]
```
cat /proc/version
Linux version 3.13.0-53-generic (buildd@phianna) (gcc version 4.8.2 (Ubuntu 4.8.2-19ubuntu1) ) #89-Ubuntu SMP Wed May 20 10:34:39 UTC 2015
```

```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.3 LTS
Release:	14.04
Codename:	trusty
```

```
uname -a
Linux cam-api-1 3.13.0-53-generic #89-Ubuntu SMP Wed May 20 10:34:39 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

@LinusU

**Comment by zbjornson** [comment]
Documentation fixed in #1056 and #1057 (see https://github.com/Automattic/node-canvas/pull/1019#issuecomment-337060038).
